Parent/Parent/Conversations/ConversationListViewController.swift

```diff
@@ -21,6 +21,7 @@ import Core
 
 class ConversationListViewController: UIViewController {
     @IBOutlet weak var activityIndicatorView: UIActivityIndicatorView!
+    @IBOutlet weak var composeButton: UIButton!
```

Comment:
I added a button class for this to reuse https://github.com/instructure/canvas-ios/pull/911/files#diff-b92eb2ec71e120917ae5b36055c17dc5

Reply:
The shadow & icon are different, plus it's named "Reply". Also it didn't exist yet. ;) Would you like me to adapt it to a more generic `FloatingButton`?

Reply:
Ah, I didn't realize the icon was different.  If my shadow is off, I think they should be same.  Also, I'd make sure it keeps the shadow path and rasterizing so it doesn't make the scrolling choppy.